packages/wonder-blocks-layout/util/test-util.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import type {MediaQuery, MediaSize} from "./types.js";
+
+/**
+ * Helper function for setting the window size, for use in tests.
+ * @param {MediaSize} size
+ */
+export function resizeWindow(size: MediaSize) {
+    if (size === "large") {
+        window.innerWidth = 1200;
+    } else if (size === "medium") {
+        window.innerWidth = 800;
+    } else if (size === "small") {
+        window.innerWidth = 640;
+    }
+
+    window.dispatchEvent(new Event("resize"));
+}
```

Comment:
I think we might want tests for some of these helpers to ensure they work as we expect.

Reply:
Good idea.